Index.md

```diff
@@ -49,5 +49,6 @@
 | [Weather-App](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/Weather-App)                                   | Simple Weather app written in HTML, CSS, and JavaScript using the APIs for fetching weather and geolocation information                                                                                                              |
 | [News Webapp](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/flask_news_project)                            | A web app for searching news made using flask                                                                                                                                                                                        |
 | Score Keeper                                                                                                                  |A web Page where user can keep a count of their scores and will tell which player won.| 
-| [Image Filter App]()                                                                                                                  |An Awesome Image Filter App written in HTML, CSS, JavaScript and CamanJS.|
-  
+| [Image Filter App]()    

```

Comment:
You have removed the Image filter app's description.

Reply:
Maam it was like that. I just clicked on new line